python/cpp/pystrings.cpp

```diff
@@ -20,9 +20,9 @@
 #include <stdio.h>
 #include <exception>
 #include <stdexcept>
-#include "NVStrings.h"
-#include "util.h"
-#include "ipc_transfer.h"
+#include <nvstrings/NVStrings.h>
```

Comment:
This should not have the <> brackets.

Reply:
These files need to be in the Python setup context in order to be built as extensions as part of the Python setup.py process. The previous way the setup.py was written was not correct and did not work with `pip install` properly. This leaves us a few options:
- Move the files into the `python` folder so they're part of the build process and rely on the installed headers
- Duplicate necessary headers into the python setup context and move it back to `#include "util.h"` instead of `#include <nvstrings/util.h>`
- Move the python setup.py to the top level directory so that the `cpp` folder is in the setup context.

Reply:
@davidwendt and I talked offline, and decided the best path forward for now is to continue in the path I'm going, but `util.h` will need to be reworked / fixed in the future.